src/Decoder/EntryTransactionDetail.php

```diff
@@ -106,7 +106,9 @@ protected function addRelatedParty(DTO\EntryTransactionDetail $detail, SimpleXML
 
         $xmlRelatedPartyName = (isset($xmlPartyDetail->Nm)) ? (string) $xmlPartyDetail->Nm : null;
         $relatedPartyType = new $relatedPartyTypeClass($xmlRelatedPartyName);
+        $orgId = $xmlRelatedPartyType?->Id?->OrgId?->Othr?->Id;
 
+        $relatedPartyType->setOrgId((string)$orgId);
```

Comment:
Model accept `null`, but here you cast `null` into `""`. So the model will never be `null`. IMHO this is wrong and the you should never set an orgId if it is non-existent.